api/v1alpha1/perconaservermysql_types.go

```diff
@@ -496,7 +497,7 @@ func (cr *PerconaServerMySQL) CheckNSetDefaults(serverVersion *platform.ServerVe
 	cr.Spec.MySQL.reconcileAffinityOpts()
 	cr.Spec.Orchestrator.reconcileAffinityOpts()
 
-	if oSize := int(cr.Spec.Orchestrator.Size); (oSize < 3 || oSize%2 == 0) && oSize != 0 && !cr.Spec.AllowUnsafeConfig {
+	if oSize := int(cr.Spec.Orchestrator.Size); cr.OrchestratorEnabled() && (oSize < 3 || oSize%2 == 0) && oSize != 0 && !cr.Spec.AllowUnsafeConfig {
```

Comment:
@pooknull thanks for PR but we need to add one more condition (if we started to work on it). 
We should not allow starting more than 1 MySQL pod if the user disabled orc even if the unsafe flag was set. And we need to print the message about it.
1. user wants to disable orc on the existing cluster operator should print the message and not do anything 
2. user wants to start a fresh cluster without orc but MySQL size is more than > 1  operator should not start it at all and print the message about it + CR should have an error state